docs/source/ide_basics.rst

```diff
@@ -0,0 +1,131 @@
+--------------------- ---------- ---------------------------------- -------------------------------------
+8/40g Accelerometer   80         S-RD, S-DD, S-ED, S-D40, S-D8      X-, Y-, Z-axis Acceleration
+--------------------- ---------- ---------------------------------- -------------------------------------
+IMU Gyroscope         47         All [1]_                           X-, Y-, Z-axis Rotation
+--------------------- ---------- ---------------------------------- -------------------------------------
+Absolute Orientation  65         All [1]_                           X-, Y-, Z-, W-axis Quaternion; Acc
+--------------------- ---------- ---------------------------------- -------------------------------------
+Relative Orientation  70         All [1]_                           X-, Y-, Z-, W-axis Quaternion
+--------------------- ---------- ---------------------------------- -------------------------------------
+MPL3115               36         All [1]_                           Pressure, Temperature [2]_
+--------------------- ---------- ---------------------------------- -------------------------------------
+MS8607                59         All [1]_                           Pressure, Temperature, Humidity [3]_
+--------------------- ---------- ---------------------------------- -------------------------------------
+SI1133                76         All [1]_                           Lux, UV
+===================== ========== ================================== =====================================
+
+.. [1] excluding early SSC/SSS/SSX models
+.. [2] 1 Hz Internal Measurements
+.. [3] 10 Hz Control Pad Measurements
+
+
+To simply use all recording data, we can iterate through each subchannel in a dataset like so::
+
+    for ch in ds.channels.values():
+        for sch in ch.subchannels:
+            print(sch)
+
+.. _eventarray_desc:
+
+EventArrays & raw data
+~~~~~~~~~~~~~~~~~~~~~~
+
+Each :class:`Channel` and :class:`SubChannel` object has a ``getSession()`` method, which returns an :class:`EventArray` object. :class:`EventArray` is a wrapper around a channel's underlying recording data that loads data on demand from the source file. You can index an :class:`EventArray` (e.g., ``eventarray[i]`` for some index ``i``) to get a numpy ``ndarray`` of data. Data is organized in an n-dimensional array.
+
+For subchannels, this will always be a 2-by-n array, where n is the number of samples recorded; ``eventarray[1]`` indexes the samples, ``eventarray[0]`` indexes the respective timestamps.
+
+For channels, this will be a (c+1)-by-n array, where n is the number of samples recorded and c is the number of subchannels; ``eventarray[1:]`` indexes the samples, ``eventarray[0]`` indexes the respective timestamps.
+
+
+Getting metadata
+----------------
+
+:class:`Dataset` makes available some basic metadata. Some useful pieces of information are stored directly as members:
+
+>>> ds.filename
+u'C:\\Users\\Public\\SSX09546_019.IDE'
```

Comment:
It might be better to use the same os-agnostic filename as before (`"/path/to/your/file.IDE"`), since you'd only see that sort of path running Windows.

In any case, the `u` prefix should probably get removed here and elsewhere, since it's a Python 2 thing.

Reply:
> the `u` prefix should probably get removed
agreed, good catch.

re: the file path itself, I kind of like the idea of this being a real-looking file path; how about `'/home/enDAQ/recordings/SSX09546_019.IDE'`? (A linux machine *could* print this for a user `enDAQ`.)

Reply:
I don't think it's necessary to use a full path... Last I looked, unlike the Lab, `idelib` never actually does anything to the path. You'll get exactly what you used to import the file.

Doesn't really matter, though... I'm happy with anything without "C:" and backslashes.

Reply:
okay cool, then if it's all the same to you I'll just leave it as it is